### ISSUE TITLE
Release mode runtime error 

### ISSUE THREAD

**Issue by sajadblog** [jkriege2/JKQtPlotter#141]
Hi,

First of all, thank you for your tremendous effort in developing an excellent plotting tool. I’ve been using version 4.0.3 in my application, and it worked flawlessly in both release and debug modes.

Last week, I switched to the latest version on the main branch to take advantage of the MultiAxis graph and ColoredBar chart. While everything compiles and executes correctly in debug mode, I’m encountering a runtime error in release mode. The error seems to originate from within the Qt core library. It’s intermittent—sometimes the application runs fine, and other times, it fails at startup.

It seems like this issue might be related to some initialization that’s occasionally missing, as the error only occurs during startup. Once the application is running, everything works as expected.

I’ve attached two images related to the error. In most cases, one of the errors in the images occurs, though on rare occasions, it runs normally.

There is only one thread in the application. 

![Error1](https://github.com/user-attachments/assets/3c1b7435-a2d8-49e6-9d89-a6548127a852)
![Error2](https://github.com/user-attachments/assets/50682b5a-54a1-48dc-84a4-b877c91e30ae)

**Comment by jkriege2** [comment]
Hmmm ... hard to say ... I played around a bit with adress-sanitizer (the MSVC flavour), but couldn't find any issues ...  other than that I never encountered such problems ...

Is your application very complex? Could there be any other source of heap corruption?

**Comment by jkriege2** [comment]
Could you use adress sanitizer on your application?

**Comment by sajadblog** [comment]
Yes, it's really hard to pinpoint the issue. I tried ASan, and while there were no errors during compilation, it consistently shows this exception at runtime.

If I am the only one facing this issue, I’d suspect something unique to my application. However, since this is the only third-party library where the problem occurs, I’d greatly appreciate it if you could let me know if you have any suspicions about the cause.

![image](https://github.com/user-attachments/assets/55cecc36-a704-4458-9b83-cb12185839a0)

**Comment by jkriege2** [comment]
Is this still relevant?
Have you tested wit the 5.0.0 beta (https://github.com/jkriege2/JKQtPlotter/releases/tag/v5.0.0-beta1)?
